common/service/sensor/sdr.c

```diff
@@ -10,8 +10,6 @@ SDR_INFO sdr_info;
 static uint16_t RSV_ID[2] = { 0 };
 bool is_sdr_not_init = true;
 
-SDR_Full_sensor full_sdr_table[MAX_SENSOR_SIZE];
```

Comment:
Technically this could have stayed here since the MAX_SENSOR_SIZE was defined in plat_sdr_table.h and that is included by this source.

Reply:
Can we move it back and just move the MAX_SENSOR_SIZE definition?

Reply:
Hi @GoldenBug ,
Thanks for your suggestion, we move the table back to common code and move the MAX_SENSOR_SIZE definition to the platform.